modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java

```diff
@@ -207,17 +211,22 @@ private DTOProperty _getDTOProperty(ObjectField objectField) {
 	}
 
 	private Response _getOpenAPI(
-			boolean addRelatedSchemas, String type, UriInfo uriInfo)
+			boolean addRelatedSchemas, ObjectDefinition objectDefinition,
+			String type, UriInfo uriInfo)
 		throws Exception {
 
+		if (objectDefinition == null) {
+			objectDefinition = _objectDefinition;
```

Comment:
This is no longer necessary as we would not provide an objectDefinition. The ObjectEntryOpenAPIResourceImpl has its own objectDefinition. (see comment on previous commit)